modules/plotting/src/interaction/boxselectioninteractionhandler.cpp

```diff
@@ -83,14 +83,18 @@ void BoxSelectionInteractionHandler::invokeEvent(Event* event) {
                 // Click in empty space
                 switch (dragRectSettings_.getMode()) {
                     case BoxSelectionSettingsInterface::Mode::Selection: {
-                        // selection changed
-                        std::vector<bool> selected(xAxis_->getSize(), false);
-                        selectionChangedCallback_.invoke(selected, append);
+                        if (xAxis_) {
```

Comment:
Why do we only use xAxis_ and not also yAxis_?

Reply:
I have don't know, It just crashed if the xAxis was nullptr...